src/Sulu/Component/SmartContent/Orm/DataProviderRepositoryTrait.php

```diff
@@ -58,21 +58,23 @@ private function findByFiltersIds($filters, $page, $pageSize, $limit, $locale, $
     {
         $parameter = [];
 
-        $queryBuilder = $this->createQueryBuilder('c')
-            ->select('c.id')
-            ->distinct()
-            ->orderBy('c.id', 'ASC');
+        $queryBuilder = $this->createQueryBuilder('c');
+
+        $parameter = array_merge($parameter, $this->appendSelect($queryBuilder, 'c', $locale, $options));
```

Comment:
Parameter will always be an empty array here, right? Why don't we remove the above variable then, and initialize it here? Then we also would not need to use `array_merge`.